internal/proto/plan.proto

```diff
@@ -181,6 +187,7 @@ message VectorANNS {
   Expr predicates = 3;
   QueryInfo query_info = 4;
   string placeholder_tag = 5;  // always be "$0"
+  VectorType vector_type = 6;
```

Comment:
`is_binary` seems useless, need to deprecate.

Reply:
Get.